scripts/greedy_ta.py

```diff
@@ -2,70 +2,73 @@
 
 # Created by Indraneel on 01/03/22
 
+from task_commander import TaskCommander
 import rospy
 from robosar_task_allocator.Environment import Environment
 from robosar_task_allocator.Robot import Robot
 from robosar_task_allocator.TA import *
 from robosar_task_allocator.task_transmitter.task_tx_move_base import TaskTxMoveBase
 import numpy as np
-import pickle
-import rospkg
+import argparse
 from actionlib_msgs.msg import GoalStatus
 
 
-rospack = rospkg.RosPack()
-maps_path = rospack.get_path('robosar_task_generator')
-package_path = rospack.get_path('robosar_task_allocator')
+class GreedyCommander(TaskCommander):
 
-def pixels_to_m(pixels):
-    return [pixels[0]*0.1,pixels[1]*0.1]
+    def __init__(self):
+        super().__init__()
+        self._graph_name = rospy.get_param("graph_name")
 
+    def pixels_to_m(self, pixels):
+        return [pixels[0]*0.1, pixels[1]*0.1]
 
-def greedy_allocator():
-    rospy.init_node('task_allocator_greedy', anonymous=True)
-    
+    def execute(self):
+        # Set up
+        nodes = np.load(self.maps_path+"/outputs/willow-full_lean.npy")
+        #filename = maps_path+'/maps/willow-full.pgm'
 
-    # Set up
-    n = 20
-    make_graph = False
-    nodes = np.load(maps_path+"/outputs/willow-full_lean.npy")
-    #filename = maps_path+'/maps/willow-full.pgm'
+        # Create robots
+        print(nodes[0])
+        robot0 = Robot(0, nodes[0], 0)
+        robot1 = Robot(1, nodes[0], 0)
+        robot2 = Robot(2, nodes[0], 0)
```

Comment:
Why are the robots hardcoded here again?

Reply:
this is deprecated, I'll fix it later if we ever want to use greedy again